io.github.realmazharhussain.GdmSettings.yaml

```diff
@@ -0,0 +1,42 @@
+app-id: io.github.realmazharhussain.GdmSettings
+command: gdm-settings
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: "42"
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --talk-name=org.freedesktop.Flatpak
+  - --filesystem=host-os
+  - --filesystem=host-etc
```

Comment:
why does it need these filesystem permission and why does it need to talk to the flatpak dbus?

Reply:
So this application is kind of a system application.
## Flatpak DBus
It needs to run a script it creates with escalated privileges on the host.
## --filesystem=host-os
It needs access to the system themes which are located in `/usr/share` and `/usr/local/share` on the host and it also needs access to `/usr/share/gnome-shell` directory.
## --filesystem=host-etc
It needs access to `/etc/dconf/db/gdm.d` and `/etc/dconf/profile/gdm`. Now that I think about it, this permission can be minimized. I'm gonna do it. Wait a second.

Reply:
Done
`--filesystem=host-etc` replaced with specific permissions
```
--filesystem=/etc/dconf/profile/gdm
--filesystem=/etc/dconf/db/gdm.d
```